src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Build/LanguageServiceErrorListProvider.cs

```diff
@@ -78,7 +78,7 @@ private async Task<AddMessageResult> AddMessageCoreAsync(TargetGeneratedError er
                 return TaskResult.False;
             });
 
-            return handled ? AddMessageResult.HandledAndStopProcessing : AddMessageResult.NotHandled;
+            return handled ?? false ? AddMessageResult.HandledAndStopProcessing : AddMessageResult.NotHandled;
```

Comment:
I'm confused by this. It says, "If `handled` is not null, return `handled`. If `handled` is null, run the ternary. The ternary asks, "I'm always `false`, so return `AddMessageResult.NotHandled`".
This code is equivalent to:
```suggestion
            return handled ?? AddMessageResult.NotHandled;
```